Bottom: df0f2e4a9ae5a3c826be1c4f4a507ea9b9bcff36
Top:    b0bf9db150afe77a15f71d00d30736fbc22ac2a4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:03:25 +0000

Remove debug from outline generation


---

diff --git a/src/polygon.c b/src/polygon.c
index c297c86..a892dce 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -105,7 +105,7 @@ dicer output is used for HIDs which cannot render things with holes
 #endif
 
 
-#define DEBUG_CIRCSEGS
+#undef DEBUG_CIRCSEGS
 
 #define ROUND(x) ((long)(((x) >= 0 ? (x) + 0.5  : (x) - 0.5)))
 
@@ -2219,18 +2219,19 @@ struct clip_outline_info {
 
 #define ROUTER_THICKNESS MIL_TO_COORD (10)
 //#define ROUTER_THICKNESS MIL_TO_COORD (0.1)
-
+#undef ROUTER_THICKNESS
+#define ROUTER_THICKNESS (line->Thickness)
 static int
 arc_outline_callback (const BoxType * b, void *cl)
 {
-  ArcType *arc = (ArcType *)b;
+  ArcType *line = (ArcType *)b;
   struct clip_outline_info *info = cl;
   POLYAREA *np, *res;
 
 #ifdef DEBUG_CIRCSEGS
-  if (!(np = ArcPoly (arc, arc->Thickness)))
+  if (!(np = ArcPoly (line, line->Thickness)))
 #else
-  if (!(np = ArcPoly (arc, ROUTER_THICKNESS)))
+  if (!(np = ArcPoly (line, ROUTER_THICKNESS)))
 #endif
     return 0;
 
diff --git a/src/polygon1.c b/src/polygon1.c
index 878b4e7..d18070f 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -115,7 +115,7 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 //#define DEBUG_JUMP
 //#define DEBUG_GATHER
 #undef DEBUG_ANGLE
-//#define DEBUG
+#undef DEBUG
 #ifdef DEBUG
 #define DEBUGP(...) pcb_fprintf(stderr, ## __VA_ARGS__)
 #else
